src/app/course/course-list/course-list.component.html

```diff
@@ -1 +1,8 @@
-<p>course-list works!</p>
+<ul class="course__list">
+  <app-course-item *ngFor="let course of courseList; let i = index; trackBy: trackByFn"
+  [course]="course"
+  (deleteCourseEmitter)="deleteCourse($event)"

```

Comment:
According to [Angular naming convention](https://angular.io/guide/styleguide#dont-prefix-output-properties) event handlers should have the `on` prefix. It's better to rename the component methods
```html
(deleteCourseEmitter)="onDeleteCourse($event)"
```